app/Http/Controllers/Api/V1/Lifestyle/MemoriesController.php

```diff
@@ -36,15 +36,20 @@ public function create()
     public function store(Request $request)
     {
         $request->validate([
-            'image' => 'required'
+
+            'image' =>  'required|url'
+            
         ]);
```

Comment:
use form request instead validating the request inside the controller to maintain code separation of concern!